sound/soc/sof/amd/acp.h

```diff
@@ -91,9 +91,9 @@ struct dma_descriptor {
 /* Scratch memory structure for communication b/w host and dsp */
 struct  scratch_ipc_conf {
 	/* DSP mailbox */
-	u8 sof_out_box[512];
+	u8 sof_out_box[1024];
 	/* Host mailbox */
-	u8 sof_in_box[512];
+	u8 sof_in_box[1024];
```

Comment:
this looks like an ABI change, if someone updates their kernel and not the firmware, how would this work?

Reply:
@reddysujith yes, if the corresponsing FW PR for rembrandt support increases the FW ABI, you will need to do the same in the kernel. Also, you'll need to check the FW ABI and keep the original size for older ABI.

Reply:
In the next patch we are using window info from firmware ,that will solve the problem.if now indow information is available we keep default to 512.